ToDoList-Redux/Binders/CheckmarkCellViewModel.swift

```diff
@@ -10,37 +10,17 @@ import Foundation
 import SwiftRex
 
 extension CheckmarkCellView {
-    
-
     // MARK: - STATE
     struct State: Equatable {
-        let id: String
```

Comment:
It's funny but now id is no longer needed. On the place where we receive the view action we do have the id already. This is a nice way to avoid inconsistent state and bugs.